DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/CompletionSummaryCardViewModelTests.cs

```diff
@@ -0,0 +1,199 @@
+﻿namespace DigitalLearningSolutions.Web.Tests.ViewModels.LearningMenu
+{
+    using System;
+    using DigitalLearningSolutions.Web.ViewModels.LearningMenu;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    class CompletionSummaryCardViewModelTests
+    {
+        private const int CustomisationId = 1;
+        private readonly DateTime? Completed = null;

```

Comment:
Is it really useful to have this as a class variable? I'd only use class variables for tests if they:
* Will be used for every test, or nearly every test
* Need to be defined as variables. For example random numbers are useful to define as variables because otherwise it's unclear what they represent. But `null`, `true` or `false` are often better to inline 🙂

Reply:
Ah thank you for clearing that up! Sorted in 9948a2d 🙂